firmware/config/boards/microrusefi/board_configuration.cpp

```diff
@@ -132,6 +132,18 @@ static void setupEtb() {
 	engineConfiguration->etb_use_two_wires = false;
 }
 
+static void setupStepperIdle() {
+	// TLE8888 two bridge drivers for stepper
+	engineConfiguration->etbIo[0].directionPin1 = TLE8888_PIN_21;
+	engineConfiguration->etbIo[0].directionPin2 = TLE8888_PIN_22;
+	engineConfiguration->etbIo[1].directionPin1 = TLE8888_PIN_23;
+	engineConfiguration->etbIo[1].directionPin2 = TLE8888_PIN_24;
```

Comment:
once #1392 merges, you can change these to set pins in `etbIo2`, and not set the `use_two_wires` bit